airflow-core/tests/unit/jobs/test_scheduler_job.py

```diff
@@ -3423,7 +3423,7 @@ def test_verify_integrity_if_dag_not_changed(self, dag_maker, session):
         session = settings.Session()
         orm_dag = dag_maker.dag_model
         assert orm_dag is not None
-        SerializedDagModel.write_dag(dag, bundle_name="testing")
```

Comment:
The entire class already operates on serialized dag already. This code here was double serializing it